test-data/unit/check-functions.test

```diff
@@ -2376,3 +2376,11 @@ def Y(x: Optional[str] = X(x_in)): ...
 
 xx: Optional[int] = X(x_in)
 [out]
+
+[case testNoComplainNoneReturnFromUntyped]
+def foo() -> None:
+    pass
+
+def lol():
+    1+'uh'
```

Comment:
Maybe add spaces around `+` (totally up to you).

Reply:
Oh actually this should just not be there at all